libnavui-speedlimit/src/main/java/com/mapbox/navigation/ui/speedlimit/model/UpdateSpeedLimitValue.kt

```diff
@@ -0,0 +1,12 @@
+package com.mapbox.navigation.ui.speedlimit.model
+
+import com.mapbox.navigation.base.speed.model.SpeedLimitSign
+import com.mapbox.navigation.base.speed.model.SpeedLimitUnit
+import com.mapbox.navigation.ui.base.formatter.ValueFormatter
+
+class UpdateSpeedLimitValue(
```

Comment:
Should this have an internal constructor? The value classes that are used with Expected have internal constructors.  Seems like they should be consistent.  This doesn't have to be addressed in this PR but we should discuss this among the team.

Reply:
@abhishek1508 thanks for getting this across the finish line.